docs/azure_jumpstart_ag/contoso_supermarket/freezer_monitor/_index.md

```diff
@@ -4,4 +4,261 @@ weight: 100
+
+#### Confirm the issue in Azure Data Explorer
+
+- Open the __Freezer Monitoring__ dashboard in ADX: https://dataexplorer.azure.com/dashboards
+
+  ![Dashboard showing all freezers](./img/adx_freezer_dashboard_default.png)
+
+- The charts are quite busy so let's make it easier to see the problem in Chicago by filtering the dashboard to show only data for the __Chicago__ store.
+
+  ![Dashboard showing filtering for Chicago](./img/adx_freezer_dashboard_select_chicago.png)
+
+- The store manager didn't tell you which freezer was having problems, but if you look at the dashboard below it's obvious that Freezer-2 (aka _device-id:Freezer-2-Chicago:Temperature_F_) is regularly exceeding the safe threshold of 20°F as indicated by the green dashed line. You can send this information to the store manager via a screenshot, or you can show the manager how she can use the Grafana dashboard to see see the same data.
+
+  ![Dashboard showing only Chicago freezers](./img/adx_freezer_dashboard_show_chicago.png)
+
+- While you're viewing the dashboard, take a look at the Seattle store to see if there are any issues that should be reported to that store manager.
+
+#### View the data in Grafana at the store
+
+From the Client VM:
+
+- Open the Edge browser, expand Grafana in the Favorites Bar, and select __Grafana Prod__
+- Login with the username _admin_ and the Windows Administrator password provided when you created the deployment
+- Click the hamburger menu next to __Home__ then click __Dashboards__
+
+  ![Grafana showing the Dashboards menu](./img/grafana_click_dashboards.png)
+- Click __General__ to see the list of dashboards then click __Chicago - Freezer Monitoring__ to open the dashboard for Chicago
+
+  ![Grafana showing list of Dashboards](./img/grafana_click_chicago.png)
+  - Notice that __freezer2__ is showing significant variability and frequently exceeding the safe threshold of 20°F.
+
+    ![Grafana showing the Chicago dashboard](./img/grafana_chicago_dashboard.png)
+- The manager can use this dashboard directly when talking to the technician about the freezer.
+
+### Scenario 2: Send alert when freezer is too warm
+
+As the manager of the Chicago store, you can use the Grafana dashboard to see the current temperature. While taking care of other store activities, you want to be notified when the temperature exceeds the safe threshold of 15°F so you can take action immediately.
+
+__NOTE: This won't really send you email because the server is not configured to send Simple Mail Transfer Protocol (SMTP) messages. However, it will help you understand the potential options available to a store manager.__
```

Comment:
```suggestion
__NOTE: This won't really send you an email because the server is not configured to send Simple Mail Transfer Protocol (SMTP) messages. However, it will help you understand the potential options available to a store manager.__
```